### PR TITLE
add mini-vignette on multiple header rows

### PR DESCRIPTION
**Pull request by apreshill**
Hi @jennybc,

Here is a draft of the mini-vignette we talked about in issue #492. Happy to make any edits requested!

Thanks,
Alison

### CODE REVIEW

File: man/excel_format.Rd [hunk 1]
```diff
@@ -29,7 +29,7 @@ Determine if files are xls or xlsx (or from the xlsx family).
 determine format. It draws on logic from two lower level functions:
 \itemize{
 \item \code{format_from_ext()} attempts to determine format from the file extension.
-\item \code{format_from_signature()} consults the \href{https://en.wikipedia.org/wiki/List_of_file_signatures}{file signature} or "magic
+\item \code{format_from_signature()} consults the \href{https://en.wikipedia.org/wiki/List_of_file_signatures}{filesignature} or "magic
```
Comment (jennybc) [inline man/excel_format.Rd]: This off-target change comes from running `document()` and having not-the-latest version of roxygen2 (here's a whitespace bug in CRAN roxygen that's fixed in dev). You can add a commit that puts this file back in its original state.

File: vignettes/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+*.html
+*.R
```
Comment (jennybc) [inline vignettes/.gitignore]: Is there any specific reason to add this .gitignore file?

File: man/excel_format.Rd (hunk 1 above)
Comment (apreshill) [reply man/excel_format.Rd]: will do!

File: vignettes/.gitignore (hunk 2 above)
Comment (apreshill) [reply vignettes/.gitignore]: no! I believe it was inserted totally inadvertently, will revert in my branch

**Comment by apreshill** [comment]
Please let me know if this fixes things! It looks like that errant `git.ignore` file was added because I used `usethis::use_vignette("multiple-header-rows")` (I think!).

**Comment by jennybc** [comment]
Sorry, one last thing. Will you add a bullet to NEWS.md? A similar thing happened reprex, so you could do sg similar: https://github.com/tidyverse/reprex/blob/master/NEWS.md

**Comment by apreshill** [comment]
Got it- I also followed Nick's example and added a short pointer at the end of the vignette, hope that is ok!

**Comment by jennybc** [comment]
Thanks!

**Pull request merged**